pmpro-sponsored-members.php

```diff
@@ -1722,9 +1722,52 @@ function pmprosm_profile_fields_seats($user)
 add_action('edit_user_profile', 'pmprosm_profile_fields_seats');
 
 function pmprosm_display_sponsored_accounts($member_ids) {
-
-    //make sure we have something to display
-	if ( empty( $member_ids) ) return '';
+        $request = $_REQUEST;
+        /*
+         * Get the current logged in user for the front end
+         * or
+         * Get user id for editing the user-edit.php page backend
+         */
+        $user_id = get_current_user_id();
+        if(isset($request['user_id'])){
+            $user_id = $request['user_id'];

```

Comment:
Need to sanitize all data coming from $_REQUEST.

Here, we should use the intval function.